src/Files.App/Data/Contracts/IWindowsWallpaperService.cs

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) 2024 Files Community
+// Licensed under the MIT License. See the LICENSE.
+
+namespace Files.App.Data.Contracts
+{
+	public interface IWindowsWallpaperService
+	{
+		bool SetDesktopWallpaper(string szPath);
```

Comment:
What does `sz` stand for?

Reply:
zero-terminated string > sz
It's Hungarian notation used in C.
I use it in methods that use CsWin32 to avoid conflicting of naming conventions according to our coding guidelines.

Reply:
Also 'a' means array > asz.